### PR TITLE
Apply default styles from cyberduck.io.

### PR DESCRIPTION
**Pull request by dkocher**

### CODE REVIEW

File: _static/css/custom.css [hunk 1]
```diff
@@ -1,4 +1,6 @@
-.wy-nav-content { 
+@import url("https://cdn.cyberduck.io/css/custom.css");
```
Comment (AliveDevil) [inline _static/css/custom.css:1]: Is there a reason to import the stylesheet from an external resource?

Comment (dkocher) [reply _static/css/custom.css:1]: Only needs to be maintained once.

**Review by AliveDevil — COMMENTED** [review]
Still don't see the benefit for this stylesheet being included. Currently nothing on the docs template uses these styles.
This would make sense if the websites (duck.sh, cyberduck.io, mountainduck.io) header is included as well.

**Comment by dkocher** [comment]
> Still don't see the benefit for this stylesheet being included. Currently nothing on the docs template uses these styles.
> This would make sense if the websites (duck.sh, cyberduck.io, mountainduck.io) header is included as well.

The font will be used.

**Review by AliveDevil — APPROVED** [review]

**Pull request merged**